app/bundles/CoreBundle/Assets/js/1.core.js

```diff
@@ -3251,15 +3259,54 @@ var Mautic = {
                     xAxes: [{
                         stacked: false,
                         ticks: {fontSize: 9},
-                        gridLines: {display:false}
+                        gridLines: {display:false},
+                        barPercentage: 35,
+
```

Comment:
The comma at the end will make problem in ES3. I know it will work in modern browsers, but there is no reason to have it there.